### DISCUSSION TITLE
Consider merging cluster and config directories

### DISCUSSION THREAD

**Discussion by keyvaann** [RADAR-base/RADAR-K8s-Infrastructure#16] [General]
I'm not sure what is the best practice but since we have made extra components optional, I think merging the 2 directories would be useful to have all of the code in one place and managing state and the variables easy as well.

**Comment by baixiac** [comment]
One concern is users could be overwhelmed by the creations of optional resources, which may also expand in the future. What we want is to provide a minimum clean stack and make them able to run `helmfile sync` without too many distractions. Let's move this to Discussions.

**Comment by keyvaann** [comment]
Completely forgot about this discussion. 
If those optional resources are disabled by default, why would the users be overwhelmed by them?
